### DISCUSSION TITLE
Hi! Having some trouble following instructions to install Mujoco1.5. Could I have some help with this?

### DISCUSSION THREAD

**Discussion by alvinli6** [google-deepmind/mujoco#533] [Asking for Help] [answered]
I'm completely new to using Github, sorry if this is the wrong place to ask for help. 

I'm trying to follow the instructions here: https://neptune.ai/blog/installing-mujoco-to-work-with-openai-gym-environments

When I run, "pip3 install -U 'mujoco-py<1.50.2,>=1.50.1'", I see a long error that I'm not sure how to fix:
![image](https://user-images.githubusercontent.com/82617252/196014074-83581d84-9ba7-4855-91c5-65012506b1ff.png)

**Comment by rohanpsingh** [comment, marked as answer]
Indeed, this is probably not the best place to ask :slightly_smiling_face: , for 2 reasons:
1. Your question is about OpenAI's [mujoco-py](https://github.com/openai/mujoco-py) which is different from DeepMind's [python bindings for MuJoCo](https://mujoco.readthedocs.io/en/latest/python.html). It is recommend to [migrate](https://mujoco.readthedocs.io/en/latest/python.html#migration-notes-for-mujoco-py) your code to the latter since those are maintained, officially supported and well-documented.
2. I think the maintainers want Q&As to be moved from Discussions to [Issues](https://github.com/deepmind/mujoco/issues).

Sorry I don't really have any solution to your actual problem. Cheers~
